documentation/sdk/docs/getting-started/rust.mdx

```diff
@@ -0,0 +1,209 @@
+
+```toml
+[dependencies]
+iota-sdk = { branch = "develop" }
+```
+
+### Build the SDK from Source
+
+
+1. Download the repository to any directory you choose:
+
+  ```shell
+  git clone https://github.com/iotaledger/iota-sdk
+    ```
+
+2. Move to the base directory:
+
+  ```shell
+ cd iota-sdk/
+    ```
+
+3. Build the SDK by running the following:
+
+ ```shell
+cargo build
+    ```
+
+## Usage
+
+### Client
+
+To use the Client module, you simply need to create a `Client`.
+
+```rust
+use iota_sdk::client::{
+    Client,
+};
+
+let client = Client::builder()
+    .with_node('https://api.testnet.shimmer.network')? // Insert your node URL here
+    .finish()
+    .await?;
+        
+let info = client.get_info().await?;
+println!("Node Info: {info:?}")
+
+Ok(())
+```
+
+### Wallet
+
+To use the Wallet module, you need to create a `Wallet`:
+
+```rust
+use iota_sdk::{
+    client::{
+        constants::SHIMMER_COIN_TYPE,
+        secret::{stronghold::StrongholdSecretManager, SecretManager},
+    },
+    wallet::{ClientOptions, Result, Wallet},
+};
+use std::path::PathBuf;
+
+#[tokio::main]
+async fn main() -> Result<()> {
+    // Setup Stronghold secret manager.
+    let secret_manager = StrongholdSecretManager::builder()
+        .password("vault.stronghold") // A password to encrypt the stored data.WARNING: Never hardcode passwords in production code.
+        .build(PathBuf::from("vault.stronghold"))?; // The path to store the account snapshot.
+
+    let client_options = ClientOptions::new().with_node("https://api.testnet.shimmer.network")?;// The node to connect to.
+
+    // Set up and store the wallet.
+    let wallet = Wallet::builder()
+        .with_secret_manager(SecretManager::Stronghold(secret_manager))
+        .with_client_options(client_options)
+        .with_coin_type(SHIMMER_COIN_TYPE)
+        .finish()
+        .await?;
+
+    // Generate a mnemonic and store it in the Stronghold vault.
+    // INFO: It is best practice to back up the mnemonic somewhere secure.
+    let mnemonic = wallet.generate_mnemonic()?;
+    wallet.store_mnemonic(mnemonic).await?;
+
+    // Create an account.
+    let account = wallet
+        .create_account()
+        .with_alias("Alice".to_string()) // A name to associate with the created account.
+        .finish()
+        .await?;
+
+
+    Ok(())
+}
+```
+
+## What's next?
+
+### How-To Guides
+
+Once you have [installed the IOTA SDK](#install-the-iota-sdk), you can start building your application. You can find
+usage examples in this Wiki's [how-to guides](../how_tos/accounts_and_addresses/create_mnemonic.mdx).
+
+### More Examples
+
+You can use the provided code [examples](https://github.com/iotaledger/iota-sdk/tree/develop/sdk/examples) to get
+acquainted with the IOTA SDK. You can use the following command to run any example:
+
+```bash
+cargo run --example example_name --release
+```
```

Comment:
We now use:
```suggestion
```bash
cargo run --release --all-features --example example_name
```
```
everywhere